From 8ebadf7b984d7d1109d1a5e81be69ed90bd00905 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Wed, 21 Oct 2020 11:47:40 +0300
Subject: [PATCH 0829/1921] arch/arm64/boot/dts/marvell: fix compilation
 warning

Fix DTB compilation warning on wrong i2c address field

Change-Id: I9ac14522c868e849d19dc8a1131a0a5a1d85c65d
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/38325
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-crb.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-crb.dts b/arch/arm64/boot/dts/marvell/cn9130-crb.dts
index 79fc88445b31..22e04a2f3629 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-crb.dts
+++ b/arch/arm64/boot/dts/marvell/cn9130-crb.dts
@@ -134,7 +134,7 @@
 	pinctrl-0 = <&cp0_i2c0_pins>;
 	clock-frequency = <100000>;
 
-	expander0: mcp23x17@20 {
+	expander0: mcp23x17@27 {
 		compatible = "microchip,mcp23017";
 		pinctrl-names = "default";
 		gpio-controller;
-- 
2.31.1

